src/classes/Defaults/Trigger/Post/PostUpdated.php

```diff
@@ -145,6 +145,10 @@ public function merge_tags() {
 			'group'         => __( 'Updating user', 'notification' ),
 		] ) );
 
+		// add revision link tag if revisions are enabled.
+		if ( WP_POST_REVISIONS ) {
```

Comment:
```suggestion
		if ( defined( 'WP_POST_REVISIONS' ) && WP_POST_REVISIONS ) {
```

Always check if the constant was defined